code/game/objects/items/reagent_containers/autoinjectors.dm

```diff
@@ -38,13 +38,13 @@
 	name = "tricordrazine autoinjector"
 	desc = "An autoinjector loaded with 3 doses of tricordrazine, a weak general use medicine for treating damage."
 	icon_state = "autoinjector-4"
-	amount_per_transfer_from_this = 5
-	list_reagents = list(/datum/reagent/medicine/tricordrazine = 15)
+	list_reagents = list(/datum/reagent/medicine/tricordrazine = 30)
 
 /obj/item/reagent_containers/hypospray/autoinjector/combat
 	name = "combat autoinjector"
-	desc = "An autoinjector loaded with two doses of healing and painkilling chemicals. Intended for use in active combat."
+	desc = "An autoinjector loaded with 2 doses of healing and painkilling chemicals. Intended for use in active combat."
 	icon_state = "autoinjector-4"
+	amount_per_transfer_from_this = 15
 	volume = 30
```

Comment:
Can delete this; it already inherit `volume = 30` from `/obj/item/reagent_containers/hypospray/autoinjector`